app/views/assignment/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Creating New Assignment</h1>
+<h3><%= print_date(@week) %></h3>
+
+
+<%= form_for :assignment do |form| %>
+  <%= form.select :employee_id, options_for_select(@employee_arr), :include_blank => true, required: true %>
```

Comment:
Replace `options...` with `@employees.map { |employee| [employee_full_name(employee), employee.id] }`

Reply:
This is a situation where a presenter would be really nice to have:

```
form.collection_select(:employee_id, @employees, :id, :full_name, :include_blank => true, required: true)
```